sdk/appconfiguration/azure-spring-cloud-starter-appconfiguration-config/pom.xml

```diff
@@ -17,8 +17,8 @@
   <dependencies>
     <dependency>
       <groupId>com.azure.spring</groupId>
```

Comment:
So, I have this going to main at the moment, but I am thinking it will need to be a feature branch. As we will need to merge in the "Dynamic Features" feature branch before releasing.

Reply:
After we do that we can move starter up to 4.0, as in this branch feature management is still 2.x.